geoid.py

```diff
@@ -127,8 +127,13 @@ def __init__(self, name="egm2008-1.pgm"):
                 raise GeoidBadDataFile("File has the wrong length")
 
             self.headerlen = headerlen
-            self.raw = mmap.mmap(fd, fullsize, mmap.MAP_SHARED, mmap.PROT_READ)
-
+            if (os.name == "posix"):
+                #Linux-Version:
```

Comment:
There is nothing Linux specific about this. It'll work on, say, Solaris or BSD as well.
Actually, you want probably just do without this comment altogether. It's clear what the "if" is doing.